src/vs/workbench/browser/parts/editor/editorGroupView.ts

```diff
@@ -1889,6 +1925,39 @@ export class EditorGroupView extends Themable implements IEditorGroupView {
 
 	//#endregion
 
+	//#region Editor Actions
+
+	getEditorActions(): { actions: IToolbarActions; onDidChange: Event<IMenuChangeEvent>; menuDisposable: IDisposable } {
```

Comment:
I think this can be made more explicit: call it `createEditorActions` and require to pass in a `disposable: DisposableStore`. That makes sure the caller has to cleanup eventually when done using. I only realised this idea now, should have mentioned it earlier.